src/strategies/citydao-square-root/examples.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "name": "Example",
+    "strategy": {
+      "name": "citydao-square-root",
+      "params": {
+        "symbol": "CITIZEN",
+        "address": "0x7EeF591A6CC0403b9652E98E88476fe1bF31dDeb",
+        "tokenId": 42,
```

Comment:
@will-holley can this strategy be used with multiple different tokensIDs? For instance, CityDAO has 3 different tokens within the same ERC1155 contract, and the total of all of these tokens should be used.

I didn't see how this would be possible with the given implementation, but I could have missed something.